db.py

```diff
@@ -65,9 +65,11 @@ def content(urlpath, my_args):
         if result > 0:
             resp = results[0]
         else:
-            resp = "NO RESULTS FOUND" 
+            resp = "NO RESULTS FOUND"
 
     conn.close()
-    data = {"data": "TBD", "meta": "THIS IS INTENDED TO BE A PUBLIC API: https://github.com/mozilla-mobile/test-dashboard/"}
+    data = {"data": "TBD",
+            "meta": "THIS IS INTENDED TO BE A PUBLIC API:\
+            https://github.com/mozilla-mobile/test-dashboard/"}
```

Comment:
```Python
data = {
    "data": "TBD",
    "meta": "THIS IS INTENDED TO BE A PUBLIC API:\
        https://github.com/mozilla-mobile/test-dashboard/",
}
```

Reply:
yea, kinda dumb to wrap that line just for linter.  I'll `#noqa` it. thanks!